text/0000-href-to.md

```diff
@@ -0,0 +1,47 @@
+- Start Date: 2015-09-12
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes the addition of a `href-to` helper which is a lightweight alternative to the `link-to` helper.
```

Comment:
i believe this may want to be named `url-for`


given:
```js
<a href=url ></a>
```

my interpretation is: 

`a` has a attribute (or property) named `href` that takes a URL as a value.

In other words, the value of the hypertext reference is the provided url.